worker/src/handlers/post.ts

```diff
@@ -56,12 +75,16 @@ export async function handlePost(request: Request): Promise<Response> {
 async function storePayload(
   storageKey: string,
   payload: SanitizedPayload,
-  currentTimestamp: number
+  currentTimestamp: number,
+  metadata?: Metadata | null
 ) {
   await KV.put(
     storageKey,
     JSON.stringify({ ...payload, last_write: currentTimestamp }),
```

Comment:
We no longer wany last_write in the payload right? As it is in the metadata now?
```suggestion
    JSON.stringify(payload),
```

Reply:
correct